spec/controllers/users_controller_spec.rb

```diff
@@ -334,4 +334,13 @@
     end
   end
 
+  describe 'PATCH #update_notifications' do
+    it 'should update notifications settings successfully' do
```

Comment:
Maybe consider adding a separate controller/view/feature test that checks whether the user is actually redirected to the notification settings page after he clicked the “Save” button on the page. Otherwise, issue #489 would technically not be represented by a test case. :wink: